cmake/kokkos_arch.cmake

```diff
@@ -301,6 +302,16 @@ IF (KOKKOS_ARCH_A64FX)
   )
 ENDIF()
 
+IF (KOKKOS_ARCH_ARMV9_GRACE)
+  SET(KOKKOS_ARCH_ARM_NEON ON)
+  COMPILER_SPECIFIC_FLAGS(
+    COMPILER_ID KOKKOS_CXX_HOST_COMPILER_ID
+    Clang   -march=armv9-a+sve -msve-vector-bits=512
+    GNU     -march=armv9-a+sve -msve-vector-bits=512
+    DEFAULT -march=armv9-a+sve
```

Comment:
Looking at https://docs.nvidia.com/grace-performance-tuning-guide.pdf, shouldn't this rather be `-mcpu=neoverse-v2`?

Reply:
changed it. Had Matt Bettencourt check this thing.